lms/static/sass/views/_login-register.scss

```diff
@@ -6,6 +6,7 @@
 $sm-btn-google: #dd4b39;
 $sm-btn-facebook: #3b5998;
 $sm-btn-linkedin: #0077b5;
+$sm-btn-azuread: #00bcf2;
```

Comment:
@omarkhan it would be better if these definitions could go in `partials/base/variables` inside lms/static it would make it easy for custom themes to override these variables. would it be feasible to move these definitions to `partials/base/_variables.scss` in this PR ?